src/main/views/MattermostView.js

```diff
@@ -202,6 +207,22 @@ export class MattermostView extends EventEmitter {
         return WindowManager.getMainWindow.webContents;
     }
 
+    handleInputEvents = (_, input) => {
+        // Handler for pressing the Alt key to focus the 3-dot menu
+        if (input.key === 'Alt' && input.type === 'keyUp' && this.altLastPressed) {
+            this.altLastPressed = false;
+            WindowManager.focusThreeDotMenu();
+            return;
+        }
```

Comment:
we might want to review this edge-case: https://github.com/mattermost/desktop/issues/1399

Reply:
I'm actually surprised this still happens, since my code should only work on keypress for `Alt` effectively. But i'll take a look on Ubuntu.

Reply:
Can't see to reproduce that issue on this branch, so I'm assuming it's been fixed.